spec/game_saver_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'game_saver'
+require_relative './test_doubles/fake_class_double'
+require 'game'
+require 'human_player'
+require 'computer_player'
+require 'board'
+require 'display'
+require 'display_colour'
+
+
+def clear_file(filename)
+  File.open(filename, 'w') { |file| file.truncate(0) }
+end
+
+RSpec.describe GameSaver do
+  describe 'Saving an object' do
+    it 'saves a hash, with the key "TicTacToe" and value "text"' do
+      game_saver = GameSaver.new
+      filename = './spec/test_data/test.yml'
+      game_name = 'TicTacToe'
+      obj = 'text'
+
+      game_saver.save(filename, game_name, obj)
+      open_file = YAML.load_file(filename)
+
+      expect(open_file['TicTacToe']).to eq('text')
+
+      clear_file(filename)
```

Comment:
Good job cleaning the file after the test!

Reply:
Thanks!